docs/topics/js/js-interop.md

```diff
@@ -202,12 +202,53 @@ fun usingAsOperator(s: Any) = s as String
 They will be compiled accordingly:
 
 ```javascript
-function usingUnsafeCast(s) {
+function usingUnsafeCast (s) {
     return s;
 }
 
-function usingAsOperator(s) {
+function usingAsOperator (s) {
     var tmp$;
     return typeof (tmp$ = s) === 'string' ? tmp$ : throwCCE();
 }
-```
\ No newline at end of file
+```
+
+## Equality
+
+Kotlin/JS has particular semantics for equality comparison. 
+
+In Kotlin/JS, the Kotlin [referential equality](equality.md#referential-equality) operator (`===`) always translates to the JavaScript
+[strict equality](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Strict_equality) operator (`===`) when comparing strings. 
+
+The JavaScript `===` operator checks not only that two values are equal but also that
+the types of these two values are equal:
+
+ ```kotlin
+fun main() {
+    val name = "kotlin"
+    val value1 = name.substring(0, 1)
+    val value2 = name.substring(0, 1)
+
+    println(if (value1 === value2) "yes" else "no")
+    // Prints 'yes' in Kotlin/JS
+    // Prints 'no' in other platforms
+}
+ ```
+
+Because of this usage of the JavaScript `===` operator, there is no Kotlin [structural equality](equality.md#structural-equality) `==` operator 
+when comparing strings in Kotlin/JS.
+
+Also, in Kotlin/JS, the [`Byte`, `Short`, `Int`, `Float`, and `Double`](js-to-kotlin-interop.md#kotlin-types-in-javascript) numeric types 
+are all represented with the `Number` JavaScript type in runtime. In consequence, these five types are numerically equal, 
```

Comment:
I don't think we can say that "types are numerically equal". Some values of these types can be.